main_exercise1.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu May 10 15:34:22 2018
+
+@author: amyhaddad
+"""
+
+from exercise1 import print_greeting
+
+def main():
+    """Get input from user and print a greeting"""
+
+    # get input
+    user_name = input("What is your name? ")
+
+    # call function
+    user_greeting = print_greeting(user_name)
```

Comment:
So I think you're mixing the function calls with the output. `print_greeting` doesn't actually print the greeting, it generates the string that is then printed on line 21. So I'd recommend calling this function `generate_greeting`.